pkg/config/labels/instance_test.go

```diff
@@ -202,7 +203,8 @@ func TestInstanceValidate(t *testing.T) {
 func BenchmarkLabelString(b *testing.B) {
 	big := labels.Instance{}
 	for i := 0; i < 50; i++ {
-		big["topology.kubernetes.io/region"] = "some value"
+		key := fmt.Sprintf("topology.kubernetes.io/region-%d", i)
+		big[key] = "some value"
```

Comment:
I think there is a bug in the old code: it loops 50 times but uses the same key.